topics/powertoys/index.md

```diff
@@ -7,7 +7,9 @@ short_description: Microsoft PowerToys is a set of utilities for power users.
 related: microsoft
 topic: powertoys
 wikipedia_url: https://en.wikipedia.org/wiki/Microsoft_PowerToys
+logo: index.png
 ---
+![banner](https://raw.githubusercontent.com/microsoft/PowerToys/master/doc/images/overview/PT%20hero%20image.png)
```

Comment:
Is this something other topics do? If not, can we remove it?

Reply:
I changed it